std_msgs/Header header
sensor_msgs/JointState joint_state
int32 attached_obj
geometry_msgs/Transform delta_transform  # transform relative to the object's original pose when attached